packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -67,8 +67,10 @@ def format(self, record):
 
 
 def config_all_loggers():
+    the_manager: logging.Manager = logging.Logger.manager
+
     loggers = [logging.getLogger()] + [
-        logging.getLogger(name) for name in logging.root.manager.loggerDict
+        logging.getLogger(name) for name in the_manager.loggerDict
```

Comment:
did this trick work in the end?

Reply:
yes. 

Long story:

Despite having the same pylint version, I could not reproduce it in my machine. So I did the following
- logged ``pip freeze`` in gh-actions job and reproduce exactly same environ in my machine
- managed to get the same linter error
- somehow ``manager`` attribute is NOT defined in the constructor, so exploring the logging code i found it was better to rewrite it in terms of the "globals" created in the ``logging`` interface.
- figured out that this code is in TWO places ... 
- changed and worked